templates/terraform/examples/node_group_basic.tf.erb

```diff
@@ -0,0 +1,18 @@
+data "google_compute_node_types" "central1a" {
+  zone = "us-central1-a"
+}
+
+resource "google_compute_node_template" "soletenant-tmpl" {
+  name = "<%= ctx[:vars]['template_name'] %>"
+  region = "us-central1"
+  node_type = "${data.google_compute_node_types.central1a.names[0]}"
+}
+
+resource "google_compute_node_group" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['group_name'] %>"
+  zone = "us-central1-a"
+  description = "example google_compute_node_group for Terraform Google Provider"
+
+  size = 0
```

Comment:
Since this isn't a useful configuration for a user, we should add a handwritten test testing with `0` and use a real value in the example.

Reply:
:+1: